### PR TITLE
[15.0][MIG] stock_picking_report_delivery_cost: Migration to version 15.0

### PR DESCRIPTION
**Pull request by pilarvargas-tecnativa**
TT39015  @Tecnativa

@victoralmau  @chienandalu please review

ping @stefan-tecnativa

### CODE REVIEW

File: stock_picking_report_delivery_cost/tests/test_stock_picking_report_delivery_cost.py [hunk 1]
```diff
@@ -69,6 +69,6 @@ def test_carrier_price_for_report_after(self):
         picking = self.order.picking_ids
         self.assertAlmostEqual(picking.carrier_price_for_report, 0)
         move = picking.move_ids_without_package
-        move.qty_done = move.product_qty
-        picking.action_done()
+        move.quantity_done = move.product_qty
+        picking._action_done()
```
Comment (chienandalu) [inline stock_picking_report_delivery_cost/tests/test_stock_picking_report_delivery_cost.py]: Better use `button_validate()`

Comment (pilarvargas-tecnativa) [reply stock_picking_report_delivery_cost/tests/test_stock_picking_report_delivery_cost.py]: ok, changed! thank you :)

**Review by victoralmau — APPROVED** [review]
Code and functional review OK.

File: stock_picking_report_delivery_cost/tests/test_stock_picking_report_delivery_cost.py [hunk 2]
```diff
@@ -52,7 +52,7 @@ def setUpClass(cls):
     def test_carrier_price_for_report_before(self):
         delivery_wizard = Form(
             self.env["choose.delivery.carrier"].with_context(
-                {
+                **{
```
Comment (rousseldenis) [inline stock_picking_report_delivery_cost/tests/test_stock_picking_report_delivery_cost.py]:
```
self.env["choose.delivery.carrier"].with_context(
    default_order_id=self.order.id,
    default_carrier_id=self.carrier.id,
)
```

**Review by rousseldenis — CHANGES_REQUESTED** [review]

File: stock_picking_report_delivery_cost/tests/test_stock_picking_report_delivery_cost.py (hunk 2 above)
Comment (pilarvargas-tecnativa) [reply stock_picking_report_delivery_cost/tests/test_stock_picking_report_delivery_cost.py]: Much better, clearer and cleaner, thanks for the suggestion.

**Comment by pilarvargas-tecnativa** [comment]
@rousseldenis can you please update your review?

**Review by rousseldenis — APPROVED** [review]

**Pull request merged**